ElvUI_ProjectZidras/Modules/UnitFrames/Elements/HealPrediction.lua

```diff
@@ -303,6 +303,30 @@ function ZUF:UpdateHealComm(_, myIncomingHeal, otherIncomingHeal, absorb, _, has
 	local missingHealth = maxHealth - health
 	local healthPostHeal = health + myIncomingHeal + otherIncomingHeal
 
+	-- xD shit code fix to visually hide heal and absorb bar if it is empty avoiding the 1 pixel line they each have by default in that state. Can prob be placed in a better position and fixed in a much better way. 
+	-- This fix is insipred by EvlUI's default function "UpdateFillBar" in their HealComm.lua, they use "if amount == 0 then bar:Hide()" there to do the same thing, calling it on UpdateHealComm. The reason I used SetAlpha instead is
+	-- that for Zidras UpdateHealComm using :Hide() breaks bar positioning.
+
+	if healthPostHeal == health then    
+        self.myBar:SetAlpha(0)
+        self.otherBar:SetAlpha(0)
+    else
+        self.myBar:SetAlpha(1)
+        self.otherBar:SetAlpha(1)
+    end
+
+    if absorb == 0 then                    
+        healAbsorbBar:SetAlpha(0)
+        absorbBar:SetAlpha(0)
+        overAbsorb:SetAlpha(0)
+    else
+        healAbsorbBar:SetAlpha(1)
+        absorbBar:SetAlpha(1)
+        overAbsorb:SetAlpha(1)
+    end
+
```

Comment:
This would fail if you had one of the IncHeals present. Ideally, for the bandaid to work, I assume you'd have to check the Incoming value for 0 and AlphaHide the respective bar